docs/platforms/java/common/performance/instrumentation/okhttp.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: OkHttp Integration
 sidebar_order: 35
+sdk: sentry.java.okhttp
```

Comment:
This can be `io.sentry:sentry-android-okhttp` or `io.sentry:sentry-okhttp` depending on Android or not.

- https://github.com/getsentry/sentry-java/blob/8fd337bbd12b54caec15e8ab3dc0f71f02b67d72/sentry-okhttp/src/main/java/io/sentry/okhttp/SentryOkHttpInterceptor.kt#L58
- https://github.com/getsentry/sentry-java/blob/8fd337bbd12b54caec15e8ab3dc0f71f02b67d72/sentry-android-okhttp/src/main/java/io/sentry/android/okhttp/SentryOkHttpInterceptor.kt#L59

Reply:
we deprecated `sentry-android-okhttp` (see docs [PR](https://github.com/getsentry/sentry-docs/pull/9494) and [impl](https://github.com/getsentry/sentry-java/blob/256ae960c665cfd692491d4d0ca8acc41a739a2d/sentry-android-okhttp/src/main/java/io/sentry/android/okhttp/SentryOkHttpInterceptor.kt#L29)) in favor of using `sentry-okhttp` on Android as well